src/wp-includes/block-supports/typography.php

```diff
@@ -523,25 +526,36 @@ function wp_get_typography_font_size_value( $preset, $should_use_fluid_typograph
 		return $preset['size'];
 	}
 
-	// Checks if fluid font sizes are activated.
-	$global_settings     = wp_get_global_settings();
-	$typography_settings = isset( $global_settings['typography'] ) ? $global_settings['typography'] : array();
-	$layout_settings     = isset( $global_settings['layout'] ) ? $global_settings['layout'] : array();
-
-	if (
-		isset( $typography_settings['fluid'] ) &&
-		( true === $typography_settings['fluid'] || is_array( $typography_settings['fluid'] ) )
-	) {
-		$should_use_fluid_typography = true;
+	/*
+	 * Backwards compatibility since 6.5.
```

Comment:
Should be 6.6?

```
* Backwards compatibility since 6.6.
* As a boolean (deprecated since 6.6), $settings acts as an override to switch fluid typography "on" (`true`) or "off" (`false`).
```

Reply:
It should be! Thanks for catching. Copy paste malfunction 😄 I'll update

Reply:
I think I meant to write

"Backwards compatibility for < 6.6."